quickstarts/jira-errors/config.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: jira-errors-inbox
+
+# Displayed in the UI (required)
+title: Atlassian Jira for Errors Inbox
+
+# Long-form description of the quickstart (required)
+description: |
+  [Connect errors inbox to Jira (Cloud)](https://newrelic.com/blog/how-to-relic/errors-inbox-with-jira) to easily create tickets for your errors, allowing for faster collaboration and resolution. Check out our documentation to learn how to set up Jira templates, which allow you to quickly create a ticket containing error details and links directly to the stack trace and APM for quick access and resolution.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Integrated errors inbox with Atlassian Jira (cloud) to easily create tickets for your errors.
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic
+
+# Authors of the quickstart (required)
+authors:
+  - Joseph Counts
+  - New Relic
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - jira
+  - error
+  - inbox
+  - newrelic
+  - atlassian
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+
+documentation:
+  - name: How to Docs
+    url: https://newrelic.com/blog/how-to-relic/errors-inbox-with-jira
```

Comment:
Is there a link to to a docs site page we can reference here vs a blog link?

Reply:
done.